oyster/src/utils/file-processor.js

```diff
@@ -1,16 +1,30 @@
 import _ from "lodash";
```

Comment:
This is the file where all the file upload magic happens in case you'd like to review @rfornea